chain_name: xylo_larm
chain_root: HeadPitch_link
chain_tip: l_stick
marker_type: circle
marker_frame: l_tip
xylo_larm_marker_radius: 0.011

marker_transform_source: tf   # 'tf' or 'urdf' (optional; if not set, default is 'urdf')